### PR TITLE
shell: Replace `sed` with `-mindepth 1` and `cut`

### PR DESCRIPTION
**Pull request by Ambrevar**

### CODE REVIEW

**Comment by Ambrevar** [comment]
`cut` is faster and more readable than `sed`.
It is also more consistent with #779.
I can't remember why I forgot to apply the changes to the rest of the functions in #779.

File: shell/key-bindings.zsh [hunk 1]
```diff
@@ -38,8 +38,8 @@ bindkey '^T' fzf-file-widget
 
 # ALT-C - cd into the selected directory
 fzf-cd-widget() {
-  local cmd="${FZF_ALT_C_COMMAND:-"command find -L . \\( -path '*/\\.*' -o -fstype 'devfs' -o -fstype 'devtmpfs' -o -fstype 'proc' \\) -prune \
-    -o -type d -print 2> /dev/null | sed 1d | cut -b3-"}"
+  local cmd="${FZF_ALT_C_COMMAND:-"command find -L . -mindepth 1 \\( -path '*/\\.*' -o -fstype 'devfs' -o -fstype 'devtmpfs' -o -fstype 'proc' \\) -prune \
+    -o -type d -print 2> /dev/null | cut -b3- "}"
```
Comment (junegunn) [inline shell/key-bindings.zsh]: A minor nit: can you remove this new space?

Comment (Ambrevar) [inline shell/key-bindings.zsh]: Done.

**Comment by junegunn** [comment]
Thanks!

**Pull request merged**